elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/AsyncIT.java

```diff
@@ -217,9 +498,91 @@ public void jsonApiRequestTests() throws InterruptedException {
+                                resource(
+                                        type("asyncQuery"),
+                                        id("edc4a871-dff2-4054-804e-d80075cf829e"),
+                                        attributes(
+                                                attr("query", "{\"query\":\"{ book { edges { node { id title } } } }\",\"variables\":null}"),
+                                                attr("queryType", "GRAPHQL_V1_0"),
+                                                attr("status", "QUEUED"),
+                                                attr("requestId", "1001"),
+                                                attr("asyncAfterSeconds", "7")
+                                        )
+                                )
+                        ).toJSON())
+                .when()
+                .post("/asyncQuery")
+                .then()
+                .statusCode(org.apache.http.HttpStatus.SC_CREATED)
+                .body("data.id", equalTo("edc4a871-dff2-4054-804e-d80075cf829e"))
+                .body("data.type", equalTo("asyncQuery"))
+                .body("data.attributes.status", equalTo("COMPLETE"))
+                .body("data.attributes.result.contentLength", notNullValue())
+                .body("data.attributes.result.responseBody", equalTo("{\"data\":{\"book\":{\"edges\":"
+                        + "[{\"node\":{\"id\":\"1\",\"title\":\"Ender's Game\"}},"
+                        + "{\"node\":{\"id\":\"2\",\"title\":\"Song of Ice and Fire\"}},"
+                        + "{\"node\":{\"id\":\"3\",\"title\":\"For Whom the Bell Tolls\"}}]}}}"))
+                .body("data.attributes.result.httpStatus", equalTo(200))
+                .body("data.attributes.result.resultType", equalTo(ResultType.EMBEDDED.toString())).toString();
+
+    }
     /**
-     * Various tests for a GRAPHQL query as a Async Request.
+     * Test for a GraphQL query as a Async Request with asyncAfterSeconds value set to 10.
+     * @throws InterruptedException
+     */
+    @Test
+    public void graphQLRequestAsyncAfterTests() throws InterruptedException {
+
+        //Create Async Request
+        given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .body(
+                        data(
+                                resource(
+                                        type("asyncQuery"),
+                                        id("edc4a871-dff2-4054-804e-d80075cf827e"),
+                                        attributes(
+                                                attr("query", "{\"query\":\"{ book { edges { node { id title } } } }\",\"variables\":null}"),
+                                                attr("queryType", "GRAPHQL_V1_0"),
+                                                attr("status", "QUEUED"),
+                                                attr("requestId", "1001"),
+                                                attr("asyncAfterSeconds", "10")
```

Comment:
How is this test different than the last one for 7 seconds?

Reply:
I initially had 0 and 10 seconds tests. I added 3 and 7 seconds tests to them. Based on your comments now I will remove the 10 s test.